tests/webview/visual/cnx_homepage_books.py

```diff
@@ -0,0 +1,480 @@
+from selenium import webdriver
+from applitools.eyes import Eyes
+from time import sleep
+from dotenv import load_dotenv
+from applitools.eyes import MatchLevel
+import os
+from tests.utils import cnx_deploy_version
+
+load_dotenv(verbose = True)
+
+applitools_key = os.environ['APPLITOOLS_API_KEY']
+url_addr = os.environ['URL_HOME_PAGE']
+App_Name = 'CNX HOME PAGE BOOK-ICONS'
+
+class Books_Home_Page_Layout:
+
+    # Verifies visual presence of book icons on cnx home page using applitools' eyes function
+    # To see results, log in to eyes.applitools.com/app/test-results
+    # Baseline images from cnx.org taken on Feb. 22, 2019
+
+    eyes = Eyes()
+
+    # Initialize the eyes SDK and set your private API key
+    eyes.api_key = applitools_key
+
+    # Level of matching the baseline and current screenshots
+    eyes.match_level = MatchLevel.STRICT
+
+    # current cnx deploy and webview versions
+    cnx_ver = cnx_deploy_version.cnx_versions
+    cnx_web_web = cnx_deploy_version.cnx_w_web
+
+
+    try:
+
+        # Open a Chrome browser
+        driver = webdriver.Chrome()
```

Comment:
See my comment about using fixtures: https://github.com/openstax/cnx-automation/pull/540/files#r261679274

You don't need to implement driver again here because it's already provided by the pytest-selenium plugin. You simply need to refactor this like mentioned before and you will have the driver available.